server/routes/api/articles.js

```diff
@@ -19,6 +19,10 @@ articlesRouter.delete('/:slug/comments/:id/like', authorizeAccess, find, Comment
 
 articlesRouter.get('/', ArticleController.getAll);
 articlesRouter.get('/:slug', ArticleController.getOne);
+
+articlesRouter.post('/:slug/like', authorizeAccess, ArticleController.like);
+articlesRouter.delete('/:slug/like', authorizeAccess, ArticleController.unlike);
+
```

Comment:
Can an unverified user like an article? If they can please let me know.

Reply:
Thanks for pointing this out. An unverified user should not be able to like an article.